.projenrc.js

```diff
@@ -1,9 +1,16 @@
 const { clickupCdk } = require('@time-loop/clickup-projen');
+
+const bundledDeps = [];
+const peerDeps = [];
+
 const project = new clickupCdk.ClickUpCdkConstructLibrary({
   name: '@time-loop/cdk-documentdb',
-  author: 'DevOps Team',
+
   cdkVersion: '2.1.0',
   defaultReleaseBranch: 'main',
-  devDeps: ['@time-loop/clickup-projen'],
+  licensed: true,
+
+  deps: [...bundledDeps, 'multi-convention-namer'],
```

Comment:
When should we add a dependency in `bundledDeps` vs. adding it inline in deps?

Reply:
when you depend on a non-jsii library, you need to bundle it.